docs/docs/strings.md

```diff
@@ -276,4 +276,35 @@ Returns a new string with the original string repeated the given number of times
 ```cs
 "ba"+"na".repeat(2); // banana
 "la".repeat(2) + " land"; // lala land
-```
\ No newline at end of file
+```
+
+### string.isUpper() -> Boolean
+
+Returns a boolean indicating that the given string is an upper case letter.
+
```

Comment:
I'd probably put a note just explaining empty would return false for `isUpper` and `isLower`

Reply:
Updated.